src/components/Result.test.tsx

```diff
@@ -80,4 +81,51 @@ describe('Result component', () => {
     });
     expect(referenceHeading).not.toBeInTheDocument();
   });
+
+  it('shows the answers when expanding a question', async () => {
+    const user = userEvent.setup();
+    const results: ResultType[] = [
+      {
+        id: '1',
+        question: {
+          id: '1',
+          referenceLink: 'https://en.wikipedia.org/wiki/Mars',
+          type: QuestionTypes.SingleChoice,
+          text: 'Which planet is known as the Red Planet?',
+          answers: [
+            { id: 'a', correct: false, text: 'Jupiter' },
+            { id: 'b', correct: true, text: 'Mars' },
+          ],
+          remainingAttempts: 0,
+          correctlyAnswered: true,
+        },
+        selections: [{ id: 'b', correct: true, text: 'Mars' }],
+      },
+    ];
+
+    const providerProps = {
+      value: {
+        results,
+      },
+    };
+
+    customRender(<Result />, { providerProps });
+
+    // Answers are initially not visible
+    let answer1 = screen.queryByRole('radio', { name: /Mars/ });
+    let answer2 = screen.queryByRole('radio', { name: /Jupiter/ });
+
+    expect(answer1).not.toBeInTheDocument();
+    expect(answer2).not.toBeInTheDocument();
+
+    const expandBtn = screen.queryByRole('cell', { name: 'Expand' });
+    await user.click(expandBtn!);
+
+    // Answers are visible after clicking the expand button
+    answer1 = screen.getByRole('radio', { name: /Mars/ });
+    answer2 = screen.getByRole('radio', { name: /Jupiter/ });
+
+    expect(answer1).toBeInTheDocument();
+    expect(answer2).toBeInTheDocument();
```

Comment:
Here it would also be cool to check if the selected radio button is checked:

```.tsx
expect(answer1).toBeChecked();
expect(answer1).toHaveAttribute('disabled');

expect(answer2).not.toBeChecked();
expect(answer2).toHaveAttribute('disabled');
```

unfortunately the first line fails in the test setup; the `checked` attribute is never present due to its disabled state, even if the checkbox seems to appear "checked".

Reply:
This has been changed with the latest implementation. We use the `aria-description` attribute.

Reply:
This has changed with the last implementation. We use `aria-description` instead.